migrations/versions/7c8d52a17f8f_add_village_venues.py

```diff
@@ -0,0 +1,30 @@
+"""add_village_venues
+
+Revision ID: 7c8d52a17f8f
+Revises: 1012d60b8c68
+Create Date: 2022-05-07 20:47:38.189635
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = '7c8d52a17f8f'
+down_revision = '1012d60b8c68'
```

Comment:
As I've just merged #1128 you'll need to change this to `822ce04a81a2`. Probably easiest to do `./flask db downgrade` twice, then rebase/merge, make the change and upgrade 3 times.